src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java

```diff
@@ -50,7 +50,7 @@
  *
  * In this example, we count the TopN articles from a stream of page views (aka clickstreams) that
  * reads from a topic named "PageViews". We filter the PageViews stream so that we only consider
- * pages of type article, and then map the recored key to effectively nullify the user, such that
+ * pages of type article, and then map the recorded key to effectively nullify the user, such that
```

Comment:
I think "record key" was correct -- the key of the record.

Reply:
oh, shoot. I'll fix it.